lib/logger.js

```diff
@@ -112,38 +111,36 @@ class Logger {
             this._traceLogger = this._logger.child({
                 level: bunyan.TRACE
             });
-            // Set up handlers for uncaught extensions
-            this._setupRootHandlers();
+
+            this._errorHandler = (err, failedStream) => {
```

Comment:
Idem...

Reply:
Could you elaborate what you want to do here?

Reply:
Ah no, nothing, this needs to stay here.